core/loaders/module.ts

```diff
@@ -22,3 +33,19 @@ export default async function (path: string): Promise<Data> {
 
   return data;
 }
+
+function getDependencies(path: string, result: Deno.EmitResult) {
+  const { sources } = JSON.parse(result.files[`${bundle}.map`]) as SourceMap;
+  const filePath = `${fileProtocol}${path}`;
+
+  return sources
+    .filter((x) => x !== filePath && x.startsWith(fileProtocol))
```

Comment:
We remove our file from the list and any file not loaded from disk (I.E. `https://` modules are excluded from the dependency list.